site/src/css/custom.css

```diff
@@ -4,958 +4,26 @@
  * bundles Infima by default. Infima is a CSS framework designed to
  * work well for content-centric websites.
  */
-
-/* You can override the default Infima variables here. */
 :root {
-  --ifm-color-primary: #253858;
-  --ifm-color-primary-dark: #161936;
-  --ifm-color-primary-darker: #151833;
-  --ifm-color-primary-darkest: #11142a;
-  --ifm-color-primary-light: #1b1f42;
-  --ifm-color-primary-lighter: #1d2045;
-  --ifm-color-primary-lightest: #21254f;
+  --ifm-color-primary: #168050;
+  --ifm-color-primary-dark: #147348;
+  --ifm-color-primary-darker: #136d44;
+  --ifm-color-primary-darkest: #0f5a38;
+  --ifm-color-primary-light: #188d58;
+  --ifm-color-primary-lighter: #19935c;
+  --ifm-color-primary-lightest: #1da668;
   --ifm-code-font-size: 95%;
   --ifm-footer-link-hover-color: #c0cee7;
+  --ifm-hero-background-color: #253858;
+  --ifm-hero-text-color: #fff;
 }
 
-.docusaurus-highlight-code-line {
```

Comment:
I tested on Firefox and Chrome and I find the layout worse with these overrides on, especially when resizing the viewport. Most of the theme tweaks have probably broken with new Docusaurus releases (since the site started when Docusaurus once in alpha). Also I think `webkit-box-orient` and `ms-flex` are superfluous for this website.